internal/backend/logger/logger.go

```diff
@@ -46,6 +37,18 @@ func (onPanicHook) OnWrite(ce *zapcore.CheckedEntry, fs []zapcore.Field) {
 }
 
 func New(cfg *Config) (*zap.Logger, error) {
+	cfg.Zap.Level.SetLevel(zapcore.Level(cfg.Level)) // Default = 0 = Info.
+
+	if cfg.Encoding != "" {
+		cfg.Zap.Encoding = cfg.Encoding
+	}
+
+	if cfg.Zap.Encoding == "console" {
```

Comment:
this is currently the default. iiuc in your change this will active only if we explicitly set encoding to "console"?

Reply:
I added these comments to clarify the logic:
```
	// Optional override for the default encoding:
	// AK default mode = Zap production config = "json" encoding,
	// AK dev mode = Zap development config = "console" encoding.
	if cfg.Encoding != "" {
		cfg.Zap.Encoding = cfg.Encoding
	}

	// Regardless of AK mode, if the encoding is "console" (whether
	// as a default or as an override), tweak it for easier reading.
	if cfg.Zap.Encoding == "console" {
		...
	}
```

Reply:
oh i see https://github.com/uber-go/zap/blob/master/config.go#L231 now
ok